docs-lab/reference/cli.md

```
@@ -866,17 +866,19 @@ With `--json`:
 
 ```bash
 openspec status --change add-rate-limit          # checklist view
 openspec status --change add-rate-limit --json   # structured report
+openspec status --all                            # every active change
+openspec status --all --json                     # one batch report
 ```
 
-`--change` is required. Without it, status exits 1 and lists the available changes, even when only one exists:
+Use exactly one of `--change` or `--all`. Without either, status exits 1 and lists the available changes, even when only one exists:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)